layouts/shortcodes/pages/compatibility-products.html

```diff
@@ -8,47 +8,120 @@
 </style>
 
 <div class="row">
-  <div class="col-md-16">
-    <ul>
-      {{ range $.Site.Data.compatible_products.sets }}
-        <li><a href="#{{ .title | urlize }}">{{ .title }}</a></li>
-      {{ end }}
-      </ul>
+  <div class="col-md-8 col-md-offset-16" style="text-align: center;"><a class="btn btn-lg btn-secondary" href="https://github.com/eclipse-ee4j/jakartaee-platform/issues?q=is%3Aissue+is%3Aclosed+label%3Acertification">Compatibility Requests</a></div>
+</div>
+
+<!-------------------- Compatible Products Tabs start ----------------------------------->
+
+<div class="row margin-top-20 content-nav-tab-toggle">
+  <div class="visible-xs">
+    <div class="vertical-align">
+      <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#navbar-member-levels">
+        <p class="nav-label"><strong>Compatible Products</strong></p>
+        <div class="hamburger-wrapper">
+          <span class="sr-only">Toggle navigation</span>
+          <span class="icon-bar"></span>
+          <span class="icon-bar"></span>
+          <span class="icon-bar"></span>
+        </div>
+      </button>
+    </div>
   </div>
-  <div class="col-md-8" style="text-align: center;"><a class="btn btn-lg btn-secondary" href="https://github.com/eclipse-ee4j/jakartaee-platform/issues?q=is%3Aissue+is%3Aclosed+label%3Acertification">Compatibility Requests</a></div>
+</div>
+ 
+<div class="row">
+ <div class="navbar-collapse collapse" id="navbar-member-levels">
+   <ul class="nav nav-tabs solstice-tabs content-nav-tab" role="tablist">
+     <li role="presentation" class="active">
+       <a class="background-grey" href="#tab-one" data-toggle="tab" aria-controls="tab-one" role="tab">
+        Jakarta EE Compatible Products
+       </a>
+     </li>
+     {{ range $.Site.Data.compatible_products.sets }}
+     <li role="presentation" class="">
+       <a class="background-grey" href="#tab-{{ .jakartaee_version }}" data-toggle="tab" aria-controls="tab-{{ .jakartaee_version }}" role="tab">
+        Jakarta EE {{ .jakartaee_version }}
+       </a>
+     </li>
+     {{ end }}
+   </ul>
+ </div>
 </div>
 
-{{ range $.Site.Data.compatible_products.sets }}
-  {{ $title:=.title }}
-  <div class="product-list">
-    <h2 class="text-center heading-line" id="{{ .title | urlize }}"><span style="background-color:#fff;">{{ .title }}</span></h2>
+<div class="content-nav-tab-body tab-content margin-bottom-20" id="tabs-content">
+
+  <div role="tabpanel" class="tab-pane tab-pane active" id="tab-one">
+    {{/*  <h2 class="h3 margin-top-0">Jakarta EE Compatible Products</h2>  */}}
```

Comment:
We should remove commented out code before final version if its not needed.

Reply:
removed